app/controllers/maps_controller.rb

```diff
@@ -135,13 +120,13 @@ def destroy
 
   # used by leaflet to fetch corner coords of each warpable
   def images
-    warpables = []
-    @map.warpables.each do |warpable|
-      warpables << warpable
-      warpables.last[:nodes] = warpable.nodes_array
-      warpables.last.src = warpable.image.url
-      warpables.last.srcmedium = warpable.image.url(:medium)
-    end
+    warpables = @map.warpables
+    # @map.warpables.each do |warpable|
```

Comment:
@jywarren I couldn't find where these values were being used but wanted to check before I remove crucial code. Let me know if its safe to remove this.Thanks